cuvs-bench/README.md

```diff
@@ -0,0 +1,90 @@
+
+We provide images for GPU enabled systems, as well as systems without a GPU. The following images are available:
+
+- `cuvs-bench`: Contains GPU and CPU benchmarks, can run all algorithms supported. Will download million-scale datasets as required. Best suited for users that prefer a smaller container size for GPU based systems. Requires the NVIDIA Container Toolkit to run GPU algorithms, can run CPU algorithms without it.
+- `cuvs-bench-datasets`: Contains the GPU and CPU benchmarks with million-scale datasets already included in the container. Best suited for users that want to run multiple million scale datasets already included in the image.
+- `cuvs-bench-cpu`: Contains only CPU benchmarks with minimal size. Best suited for users that want the smallest containers to reproduce benchmarks on systems without a GPU.
+
+Nightly images are located in [DockerHub](https://hub.docker.com/r/rapidsai/cuvs-bench), meanwhile release (stable) versions are located in [NGC](https://hub.docker.com/r/rapidsai/cuvs-bench).
+
+## Container Usage
+
+The containers can be used in two manners:
+
+1. **Quick benchmark with single `docker run`**: The docker containers already include helper scripts to be able to invoke most of the functionality of the benchmarks from docker run for a simple and easy way to run benchmarks.
+
+For GPU systems, where $DATA_FOLDER is a local folder where you want datasets stored in $DATA_FOLDER/datasets and results in $DATA_FOLDER/results:
+
+```bash
+export DATA_FOLDER=path/to/store/results/and/data
+docker run --gpus all --rm -it \
+    -v $DATA_FOLDER:/home/rapids/benchmarks  \
+    -u $(id -u) \
+    rapidsai/cuvs-bench:24.12a-cuda12.5-py3.12 \
+    "--dataset deep-image-96-angular" \
+    "--normalize" \
+    "--algorithms cuvs_cagra" \
+    ""
+```
+
+Where:
+
+```bash
+export DATA_FOLDER=path/to/store/results/and/data # <- Results and datasets will be written to this host folder.
+docker run --gpus all --rm -it \
+    -v $DATA_FOLDER:/home/rapids/benchmarks  \ # <- local folder to store datasets and results
+    -u $(id -u) \ # <- this flag allows the container to use the host user for permissions
+    rapidsai/cuvs-bench:24.12a-cuda12.5-py3.12 \ # <- image to use, either `cuvs-bench` or `cuvs-bench-datasets`
+    "--dataset deep-image-96-angular" \ # <- dataset name
+    "--normalize" \ # <- whether to normalize the dataset, leave string empty ("") to not normalize.
+    "--algorithms cuvs_cagra" \ # <- what algorithm(s) to use as a ; separated list, as well as any other argument to pass to `cuvs_bench.run`
+    "" # optional argumetns to pass to `cuvs_benchmarks.plot`
+```
+
+For CPU systems the same interface applies, except for not needing the gpus argument and using the cpu images:
+```bash
+export DATA_FOLDER=path/to/store/results/and/data
+docker run  all --rm -it \
+    -v $DATA_FOLDER:/home/rapids/benchmarks  \
+    -u $(id -u) \ # <- this flag allows the container to use the host user for permissions
```

Comment:
This comment breaks the line continuation. See above. This must be removed for users to be able to copy-paste this command.
```suggestion
    -u $(id -u) \
```